Find a functionally equivalent code:android.media.Ringtone.setUri:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Set {@link Uri} to be used for ringtone playback. Attempts to open
 * locally, otherwise will delegate playback to remote
 * {@link IRingtonePlayer}.
 *
 * @hide
 */

@@ -9,15 +9,15 @@
     mLocalPlayer = new MediaPlayer();
     try {
         mLocalPlayer.setDataSource(mContext, mUri);
-        mLocalPlayer.setAudioStreamType(mStreamType);
+        mLocalPlayer.setAudioAttributes(mAudioAttributes);
         mLocalPlayer.prepare();
     } catch (SecurityException e) {
         destroyLocalPlayer();

